src/core/server/graph/loaders/Comments.ts

```diff
@@ -307,7 +307,8 @@ export default (ctx: GraphContext) => ({
         first: defaultTo(first, 10),
         orderBy: defaultTo(orderBy, GQLCOMMENT_SORT.CREATED_AT_DESC),
         after,
-      }
+      },
+      flatten
```

Comment:
Keeping with existing flows, I'd suggest moving the `flatten` into a filter spread as you can see above in the `forStory` loader with the `...tagFilter`.